checker/tests/nullness/Issue961.java

```diff
@@ -4,18 +4,19 @@
 
 // Test case for Issue 961
 // https://github.com/typetools/checker-framework/issues/961
-// @skip-test
 public class Issue961 {
     <T> T method(T param, Map<T, Object> map) {
         if (map.containsKey(param)) {
             @NonNull Object o = map.get(param);
             return param;
         }
-        return null;
+        return param;
     }
 
     abstract class MapContains<K, V> {
+        @SuppressWarnings("method.invocation.invalid")
```

Comment:
Add a comment that we want to allow invocation in field initializer. Why suppression instead of expected error?